recipes/harfbuzz/all/conanfile.py

```diff
@@ -154,14 +154,21 @@ def is_vs_2017():
             "tests": "disabled",
             "docs": "disabled",
             "benchmark": "disabled",
-            "icu_builtin": "false"
+            "icu_builtin": "false",
         })
+        if Version(self.version) >= "7.3.0":
+            tc.project_options.update({
+                "utilities": "disabled",
+            })
         tc.cpp_args += cxxflags
         tc.generate()
 
     def build(self):
         apply_conandata_patches(self)
-        replace_in_file(self, os.path.join(self.source_folder, "meson.build"), "subdir('util')", "")
+        if Version(self.version) < "7.3.0":
+            replace_in_file(self, os.path.join(self.source_folder, "meson.build"), "subdir('util')", "")
+        if Version(self.version) >= "8.4.0":
+            replace_in_file(self, os.path.join(self.source_folder, "meson.build"), "version: freetype_min_version,", "")
```

Comment:
This workaround shouldn't be necessary when #22974 is merged.

Reply:
@jwillikers 
Thanks a lot!
I'm waiting for it.

Reply:
Is this no longer needed then?

Reply:
@toge bump here - can this workaround be removed now that the linked PR has been merged?

Reply:
Ping @toge